middleware/authz.go

```diff
@@ -0,0 +1,41 @@
+package middleware
+
+// This plugin is based on Casbin: an authorization library that supports ACL, RBAC, ABAC
+// View source at:
+// https://github.com/hsluoyz/casbin
+
+import (
+	"net/http"
+
+	"github.com/hsluoyz/casbin"
```

Comment:
@VojtechVitek is right, we're not able to include 3rd party imports into chi. 

@VojtechVitek cool idea, we can make pressly/middleware for 3rd party middlewares. Question: if we make pressly/middleware/casabin and say pressly/middleware/<someotherthirdparty> can govendor or other things manage import paths only for the packages actually used by a project and not bloating child deps